content/sips/sip-161.md

```diff
@@ -48,72 +50,155 @@ On the topic of scaling and decentralizing the Synthetix Protocol, the onboardin
 
 <!--This is a high-level overview of *how* the SIP will solve the problem. The overview should clearly describe how the new feature will be implemented.-->
 
-- Define roadmap and priorities, handle compensation to core contributors, packages (salary and equity)
-- Enforcing resources, since they handle the compensation they ensure that the core contributors are aligned and focussed on their task at hand
-- Core contributors are able to non-bindingly signal what they think is important to be worked on out of the SIPs waiting to be done
-- CCC also vouch for new hires and vote on the new hires
-
 The Core Contributor Committee will be made up of three senior members of the Core Contributors, elected by the other Core Contributors. This committee decides (via a ⅔ majority) on a variety of decisions all within the context of:
 
-- Core Contributor Engagement
-- Core Contributor Compensation
-- Core Contributor Prioritization
+- Core Contributor Engagement/Disengagement
+- Core Contributor Incentivization
+- Core Contributor Resourcing & Prioritization
 - Protocol Road-map and Vision
+- Core Contributor Committee Stipends & Transparency
+
+#### Core Contributor Engagement/Disengagement
+
+The on-boarding of new Core Contributors will be a process of coordination between the
 
-#### Core Contributor Engagement
+- Core Contributors
+- Core Contributor Committee members
+- Treasury Council
 
-The onboarding of new Core Contributors would be handled between the existing Core Contributors and the CCC members. For a candidate to be eligible for consideration as a new Core Contributor, the candidate would need to have 3 existing Core Contributors vouch for them, once this has happened, the 3 Core Contributors who have vouched for this individual organise a meeting with the CCC and the candidate. The CCC members will then have the final say (by a super majority vote) on whether that candidate will be engaged with the Synthetix Protocol and propose a salary and/or a token package.
+For a candidate to be eligible for a review from the CCC, they will need to receive 3 vouches from existing or past Core Contributors. The CCC will then interview with the candidate and make a decision on their engagement via a proposal and vote, based on both the candidates merits and also the Treasury Council budget that the CCC operates within (the CCC at any point may interface with the Treasury Council to request a larger budget allocation).
```

Comment:
Should the number of CC approvals required to bring on another CC be up to the CCC to determine?